Gemfile.lock

```diff
@@ -125,8 +124,6 @@ PLATFORMS
   x86_64-linux
 
 DEPENDENCIES
-  cocoapods (~> 1.11)
-  cocoapods-generate (~> 1.0)
   jazzy (~> 0.13)
```

Comment:
CocoaPods still appears in Gemfile.lock because it's a transitive dependency of jazzy. Might explore switching to docc at some point.